### ISSUE TITLE
IIA-APPROVAL-RESPONSE contain V6 condition hash instead new V7 iiaHash

### ISSUE THREAD

**Issue by skishk** [erasmus-without-paper/ewp-specs-api-iias#174]
Dear Colleagues,

is it possibile to use conditionHash code of the old IIA in V6 as the new iiaHash in the IIA-APPROVAL-RESPONSE ?

**Comment by janinamincer-daszkiewicz** [comment]
No: https://github.com/erasmus-without-paper/ewp-specs-api-iias/blob/a151f579ba2278dfbb99d19f40cff18212425e1d/endpoints/get-response.xsd#L243

**Comment by demilatof** [comment]
> No:
> 
> https://github.com/erasmus-without-paper/ewp-specs-api-iias/blob/a151f579ba2278dfbb99d19f40cff18212425e1d/endpoints/get-response.xsd#L243

What could be the consequence for the providers that make this kind of error and for their partner?
I suppose that an approved IIA would seem as not approved.

**Comment by janinamincer-daszkiewicz** [comment]
Yes. Also, because conditionHash  is not based on XSLT you cannot verify XSLT v6 against XSLT v7. This will trigger the new approval process.

**Comment by skishk** [comment]
> No:
> 
> https://github.com/erasmus-without-paper/ewp-specs-api-iias/blob/a151f579ba2278dfbb99d19f40cff18212425e1d/endpoints/get-response.xsd#L243

I opened a tkt one week ago ESCI-11994 , but the problem is still in production...

**Comment by janinamincer-daszkiewicz** [comment]
I will try to intervene.

**Comment by skishk** [comment]
> > No:
> > https://github.com/erasmus-without-paper/ewp-specs-api-iias/blob/a151f579ba2278dfbb99d19f40cff18212425e1d/endpoints/get-response.xsd#L243
> 
> What could be the consequence for the providers that make this kind of error and for their partner? I suppose that an approved IIA would seem as not approved.

consequence is not really nice for who is working with hundreds of IIAs...(IROs) it's like everything with V7 is no more approved.... 
we are trying to contain this problem as is our possibilities... but we hope they will resolve quickly the problem... obviously this require extra effort for all of us... (IT and IROs) 😰

**Comment by skishk** [comment]
> I will try to intervene.

thank you very much

**Comment by demilatof** [comment]
> Yes. Also, because conditionHash is not based on XSLT you cannot verify XSLT v6 against XSLT v7. This will trigger the new approval process.

Well, I agree.
I don't need to test this aspect, because we will enter into production straightly with version 7, but the providers aren't tested to verify that they support the migration from v6 to v7?

**Comment by skishk** [comment]
> > Yes. Also, because conditionHash is not based on XSLT you cannot verify XSLT v6 against XSLT v7. This will trigger the new approval process.
> 
> Well, I agree. I don't need to test this aspect, because We will enter into production straightly with version 7, but the providers aren't tested to verify that they support the migration from v6 to v7?

No… :(
